nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/resources/PropertiesBagResource.java

```diff
@@ -189,27 +190,33 @@ protected ActionReportResult clearThenSaveProperties(List<Map<String, String>> p
 
                 // the prop name can not contain .
                 // need to remove the . test when http://java.net/jira/browse/GLASSFISH-15418  is fixed
-                boolean canSaveDesc = !((Object)property.get("name")).toString().contains(".");
+                boolean isDottedName = ((Object)property.get("name")).toString().contains(".");
```

Comment:
Can we remove this Object casting madness here? Basically every cast to Object in this method is unnecessary (also when disguised as `Object.class.cast` above).